exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function collatzConjecture() {}
+
+collatzConjecture.prototype.steps = function(no){
+	var count = 0;
+  if(no < 1) {
+    throw new Error('Only positive numbers are allowed');
+  }
+	getStepsCount(no);
+  function getStepsCount(n){
+  	if(n === 1) return 0;
+  	n = ( n % 2 === 0 ? n/2 : (3*n) + 1);
```

Comment:
The formatting here should be cleaned up a bit:
`n = n % 2 === 0 ? n / 2 : 3 * n + 1;`